app/src/main/java/org/fossasia/phimpme/gallery/activities/LFMainActivity.java

```diff
@@ -289,6 +289,9 @@ public boolean onLongClick(View v) {
                     mediaAdapter.notifyItemChanged(toggleSelectPhoto(m));
                     editMode = true;
                 }
+                else {
```

Comment:
@angmas1 just make one more change here too. Move the else line to the line above it, where the if block ends and remove the braces like in line 280.